src/node_crypto.cc

```diff
@@ -3246,6 +3247,7 @@ Local<Function> KeyObject::Initialize(Environment* env, Local<Object> target) {
   Local<FunctionTemplate> t = env->NewFunctionTemplate(New);
   t->InstanceTemplate()->SetInternalFieldCount(
       KeyObject::kInternalFieldCount);
+  t->Inherit(BaseObject::GetConstructorTemplate(env));
```

Comment:
@tniessen Just a heads up, this is where the conflicts I mentioned to you came from